configs/config.json.example

```diff
@@ -74,6 +74,7 @@
     "evolve_captured": "NONE",
     "catch_randomize_reticle_factor": 1.0,
     "catch_randomize_spin_factor": 1.0,
+    "move_to_fort_even_if_full_inventory": false,
```

Comment:
This option is added here because it is used from both `SpinFort` and `MoveToFort` tasks (by calling `PokemonGoBot#should_move_to_fort_for_loot` method).
Is there better way to share a config between tasks?